styles.css

```diff
@@ -1,1044 +1,1234 @@
+  color: #172b4d;
+}
+
+footer.socials {
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+  gap: 20px;
+}
+
+section#action-btn {
+  flex-basis: 10em;
+  margin-top: 5em;
+}
+
+button.ac-btn {
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+  background-color: transparent;
+  border: 1px solid #dfe1e6;
+}
+
+button.ac-btn:hover {
+  background-color: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+button.ac-btn:active {
+  background: #008552;
+}
+
+/*
+Projects Section
+*/
+section#projects {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: auto;
+  grid-template-areas: "pj-heading" "project-list";
+}
+
+h3.pj-heading {
+  grid-area: pj-heading;
+  text-align: center;
+  font-size: 45px;
+  margin-top: 20px;
+  position: relative;
+  padding-bottom: 30px;
+  color: #172b4d;
+}
+
+h3.pj-heading::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  right: 145px;
+  width: 70px;
+  height: 6px;
+  background-color: #36b37e;
+  border-radius: 10px;
+}
+
+section.project-list {
+  margin-top: 4em;
+  padding: 20px;
+  grid-area: project-list;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: auto;
+  row-gap: 25px;
+  grid-template-areas: "pj-1" "pj-2" "pj-3" "pj-4" "pj-5" "pj-6";
+}
+
+div.pj-card {
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+}
+
+div.pj-thumbnail {
+  height: 250px;
+}
+
+div.pj-card-body {
+  background: #fff;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  border-bottom-right-radius: 8px;
+  border-bottom-left-radius: 8px;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2.pj-card-title {
+  text-align: center;
+  font-size: 24px;
+  color: #3a4a42;
+  line-height: 32px;
+}
+
+ul.pj-card-lists {
+  padding-left: 0;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+li.pj-card-list-item {
+  background-color: #ebf0ee;
+  padding: 8px 12px;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+}
+
+li.pj-card-list-item span {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+section.pj-1 {
+  grid-area: pj-1;
+}
+
+section.pj-2 {
+  grid-area: pj-2;
+}
+
+section.pj-3 {
+  grid-area: pj-3;
+}
+
+section.pj-4 {
+  grid-area: pj-4;
+}
+
+section.pj-5 {
+  grid-area: pj-5;
+}
+
+section.pj-6 {
+  grid-area: pj-6;
+}
+
+button.pj-card-action {
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+div#proj-overlay {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  background-color: #3a4a4290;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 10;
+  display: none;
+  overflow-y: auto;
+}
+
+.proj-name {
+  grid-area: proj-name;
+  font-size: 32px;
+  font-weight: 700;
+  color: #172b4d;
+}

```

Comment:
- [ ] Good job so far 👏. In addition to what you have done, I would suggest you adjust the styles of the popup window in order to fit in the window screen without needing to scroll down to view all the content. This improves the user experience. 

This is how currently it looks 👇 
![image](https://user-images.githubusercontent.com/90222110/189213687-7ab4d7df-f4b2-45d9-ac83-840b706bbe53.png)

I really liked the way it looks on the mobile screens 😃 💯 and It will be good if it looks great on the desktop screens too.